client/e2e/poms/operation.ts

```diff
@@ -25,7 +66,95 @@ export class OperationPOM {
-    const currentUrl = await this.page.url();
-    await expect(currentUrl.toLowerCase()).toMatch(path.toLowerCase());
+    const currentUrl = this.page.url();
+    expect(currentUrl.toLowerCase()).toMatch(path.toLowerCase());
+  }
+
+  async operationFormIsVisible() {
+    await this.page.waitForSelector("form");
+    await expect(this.operationPage1Title).toBeVisible();
+
+    // Check for the presence of the multistep form headers
+    expect(
+      this.page
+        .getByTestId("multistep-header-title")
+        .getByText("Operation Information"),
+    ).toBeVisible();
+    expect(
+      this.page
+        .getByTestId("multistep-header-title")
+        .getByText("Point of Contact"),
+    ).toBeVisible();
+    expect(
+      this.page
+        .getByTestId("multistep-header-title")
+        .getByText("Statutory Declaration"),
+    ).toBeVisible();
+  }
+
+  async operationFormStep2IsVisible() {
+    await expect(this.operationPage2Title).toBeVisible();
+  }
+
+  async operationFormStep3IsVisible() {
+    await expect(this.operationPage3Title).toBeVisible();
+  }
+
+  async operationSuccessfulSubmissionIsVisible() {
+    this.page.getByText(
+      "Your application for the B.C. OBPS Regulated Operation ID for Test Operation Name has been received.",
+    );
+    this.page.getByText(
+      "Once approved, you will receive a confirmation email.",
+    );
+    this.page.getByText(
+      "You can then log back in and view the B.C. OBPS Regulated Operation ID for Test Operation Name.",
+    );
+  }
+
+  async clickCancelButton() {
+    await this.buttonCancel.click();
+  }
+
+  async clickNextButton() {
+    await this.buttonNext.click();
+  }
+
+  async clickSaveAndContinue() {
+    await this.buttonSaveAndContinue.click();
+  }
+
+  async clickSubmitButton() {
+    await this.buttonSubmit.click();
+    await this.page.waitForResponse((response) => response.status() === 200, {
+      timeout: 30000,
+    });
+  }
+
+  async fillOperationFormPage1() {
+    // Fill out the operation information form
+    await this.page.getByLabel("Operation Name*").fill("Test Operation Name");
+
+    await this.page.getByTestId("root_type").click();
+    await this.page
+      .getByRole("option", { name: "Single Facility Operation" })
+      .click();
+
+    await this.page.getByTestId("root_naics_code_id").click();
+    await this.page
+      .getByRole("option", {
+        name: "211110 - Oil and gas extraction (except oil sands)",
+      })
+      .click();
+  }
+
+  async fillOperationFormPage2() {
+    // Fill out the point of contact form
+    await this.page.getByLabel("Yes").click();
+    await this.page.getByLabel("First Name*").fill("Test First Name");
+    await this.page.getByLabel("Last Name*").fill("Test Last Name");
+    await this.page.getByLabel("Position Title*").fill("Test Position Title");
+    await this.page.getByLabel("Email Address*").fill("test@test.com");
+    await this.page.getByLabel("Phone Number*").fill("403 777 7777");
```

Comment:
Looking at the `fillAllFormFields`, ` fillRequiredFormFields`, and `triggerFormatValidationErrors` it might be out of scope and maybe a tech debt ticket is in order if the goal is to make them work for reg part 2. As it's written it's essentially hardcoded and needs a total rework to select fields by widget and not by the label name. 

I think some of them are unavoidable, for example CRA business number which uses an `rjsf` format and regex validation but `PhoneWidget`, `PostalCodeWidget` and `URLWidget` fields shouldn't be tied to the label.

Reply:
What's wrong with tying them to the label? That's the part the user's going to see, so labels are good selectors from that point of view

Reply:
@BCerki I can't seem to find your response to this, though I saw it in my email. I think tying them to the label works great in this case where we have a function specifically for this form.  
  
Making a generic `fillAllFormFields` function should not be tied to the labels since the labels can change. Another issue with this is this `fillAllFormFields` function works well with text inputs but not so much select/combobox/boolean fields. Some of these fields are expecting booleans/arrays and are MUI components with odd structures.

I've been thinking about how to do this function without adding test ids to all the widgets though I think it needs a small spike which is why I'm suggesting a tech debt card.

Reply:
Gotcha, tech debt it is